tools/picard.py

```diff
@@ -150,27 +150,22 @@ def execute(self, inBam, outFastq1, outFastq2, outFastq0=None,
         trim reads at the clipping position specified by the Illumina clipping attribute
         (which is defined by the class variable SamToFastqTool.illumina_clipping_attribute).'''
 
-        if tools.samtools.SamtoolsTool().isEmpty(inBam):
-            # Picard SamToFastq cannot deal with an empty input BAM file
```

Comment:
It seems like picard now properly handles empty input in this case - outputting an empty output.